packages/react-storage/src/components/StorageBrowser/actions/handlers/listLocationItems.ts

```diff
@@ -92,14 +92,13 @@ export const listLocationItemsHandler: ListLocationItemsHandler = async (
   const { bucket: _bucket, credentials, region } = config;
 
   const {
+    exclude: excludedType,
```

Comment:
We can just update the use of excludedType below
```suggestion
    exclude,
```